apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -471,12 +472,13 @@ export const publishDocuments = async ({ body }, response) => {
 
 	await Promise.all(activityLogs);
 
-	const publishedDocuments = await publishNsipDocuments(publishableDocumentVersionIds);
+	const publishedDocuments = await publishDocumentVersions(publishableDocumentVersionIds);
 
 	logger.info(`Published ${publishedDocuments.length} documents`);
 	response.send(
-		publishedDocuments.map(({ documentGuid, publishedStatus }) => ({
-			guid: documentGuid,
+		// @ts-ignore
```

Comment:
what' the .ts issue?

Reply:
That wasn't supposed to be left in. Just pushed another change.